src/modinfoforeign.cpp

```diff
@@ -49,7 +42,15 @@ ModInfoForeign::ModInfoForeign(const QString& modName, const QString& referenceF
     : ModInfoWithConflictInfo(core), m_ReferenceFile(referenceFile),
       m_Archives(archives), m_ModType(modType)
 {
-  m_CreationTime = QFileInfo(referenceFile).birthTime();
+  m_CreationTime          = QFileInfo(referenceFile).birthTime();
+  IPluginGame const* game = qApp->property("managed_game").value<IPluginGame*>();
```

Comment:
The game can be access from `core` here, there is no need to rely on `qApp`.

Reply:
I was being lazy and using existing code from elsewhere in the file, but you're right.